bitnami/airflow/templates/_init_containers_sidecars.tpl

```diff
@@ -60,6 +60,8 @@ Returns an init-container that prepares the Airflow configuration files for main
     {{- end }}
       airflow_conf_set "celery" "broker_url" "redis://${redis_credentials}@${REDIS_HOST}:${REDIS_PORT_NUMBER}/${REDIS_DATABASE}"
     {{- end }}
+      # Configure authentication backend
+      airflow_conf_set "core" "auth_manager" "airflow.providers.fab.auth_manager.fab_auth_manager.FabAuthManager"
```

Comment:
Is this required regardless the Airflow major version?

Reply:
It is mandatory for Airflow 3, and optional for Airflow 2 (it is the default value on 2.x source code).